### ISSUE TITLE
[a11y] VoiceView doesn't read page titles

### ISSUE THREAD

**Issue by rpappalax** [mozilla-mobile/firefox-echo-show#143]
## Steps to reproduce
1. go to device settings
2. Activate Accessibility > VoiceView
3. Open nytimes.com (or cnn.com) site on Silk
4. VoiceView reads out page title
5. Open nytimes.com (or cnn.com) on Firefox

### Expected behavior
6. VoiceView reads out page title on Firefox

### Actual behavior
6. VoiceView does not read out page title

### Device information

* Echo Show device (2nd gen, build #625531610)
* Firefox version: 1.2
